keyboards/hotdox/keymaps/bliss/rules.mk

```
@@ -0,0 +1,5 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+
+EXTRA_KEY_ENABLE = yes
```

```suggestion
EXTRAKEY_ENABLE = yes
```
This rule has a typo. If mediakeys work as is, then you don't need this line at all.